includes/widgets/image.php

```diff
@@ -318,6 +318,52 @@ protected function _register_controls() {
 			]
 		);
 
+		$this->add_responsive_control(
+			'height',
+			[
+				'label' => __( 'Height', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'default' => [
+					'unit' => 'px',
+				],
+				'tablet_default' => [
+					'unit' => 'px',
+				],
+				'mobile_default' => [
+					'unit' => 'px',
+				],
+				'size_units' => [ 'px' ],
```

Comment:
@ramiy Could you please add `vh` to the units acceptable?

Reply:
@shilo-ey, I had opened this issue to ask for this feature, if you want to add it to the milestone in question : https://github.com/elementor/elementor/issues/10962

Reply:
@arielk